tap_salesforce/sync.py

```diff
@@ -152,6 +152,11 @@ def sync_records(sf, catalog_entry, state, counter):
     LOGGER.info("Syncing Salesforce data for stream %s", stream)
 
     for rec in sf.query(catalog_entry, state):
+        replication_key_value = replication_key and singer_utils.strptime_with_tz(
+            rec[replication_key]
+        )
+        if replication_key_value < chunked_bookmark:
```

Comment:
should it be `<=` to not perform the checkpoint-day twice?

Reply:
you are right